src/main/java/com/example/napolinights/controller/MenuController.java

```diff
@@ -4,169 +4,156 @@
 import com.example.napolinights.model.MenuItem;
 import com.example.napolinights.model.MenuItemDAO;
 import com.example.napolinights.model.SqliteConnection;
+import javafx.application.Platform;
 import javafx.fxml.FXML;
+import javafx.geometry.Insets;
 import javafx.geometry.Pos;
 import javafx.scene.control.Label;
-import javafx.scene.image.ImageView;
-import javafx.scene.image.Image;
-import javafx.scene.layout.TilePane;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
 import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
 
-import java.io.InputStream;
-import java.net.URL;
 import java.sql.Connection;
+import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 
+/**
+ * Controller for the Menu Page. Handles the display of menu items
+ * categorized by type, and ensures the user interface adapts to the
+ * specified layout with scrolling enabled.
+ */
 public class MenuController {
 
+    // ScrollPane to hold the VBox for scrolling through the menu items
     @FXML
-    private VBox contentAnchorPane; // VBox to hold all categories and items
+    private ScrollPane menuScrollPane;
 
+    // VBox to dynamically add categories and their menu items inside the ScrollPane
+    @FXML
+    private VBox contentAnchorPane;
+
+    /**
+     * Initializes the MenuController. Sets up the ScrollPane and loads the menu items.
+     */
     public void initialize() {
-        // Load the categories and their items
+        // Set padding for the contentAnchorPane to provide spacing
+        contentAnchorPane.setPadding(new Insets(10, 50, 10, 50)); // Top, right, bottom, left padding
+
+        // Ensure the ScrollPane fits the width of its content
+        menuScrollPane.setFitToWidth(true);
+
+        // Load the menu items from the database
         loadMenuItems();
-        /*addCategory("Starters & Salads", Arrays.asList(
-                new MenuItem(Category.ENTREE.toString(), "Freshly cooked with mozzarella", 18.00, "path_to_image1"),
-                new MenuItem("Fried Green Olives", "Lightly crumbed and served with truffle mayo", 24.00, "path_to_image2")
-        ));
-
-        addCategory("WOODFIRED PIZZA", Arrays.asList(
-                new MenuItem("Margherita", "Traditional pizza with fresh tomato and mozzarella", 20.00, "path_to_image3"),
-                new MenuItem("Pepperoni", "Topped with spicy pepperoni and mozzarella", 22.00, "path_to_image4")
-        ));*/
+
+        // Ensure that the stage size is adjusted after the scene is loaded
+        Platform.runLater(() -> {
+            Stage stage = (Stage) menuScrollPane.getScene().getWindow();
+            stage.setMinWidth(800);
+            stage.setMinHeight(750);
```

Comment:
MinHeight is inconsistent, it 750 here but it's 600 in other places.  Do we need a comment to why it's different.